docs/source/release/v5.1.0/indirect_geometry.rst

```diff
@@ -8,7 +8,12 @@ Indirect Geometry Changes
 Improvements
 ############
 
-- An issue with IndirectDataReduction causing it to slow down processing when open has been resolved.
+.. figure:: ../../images/Indirect_Data_Analysis_5_1_0.png
```

Comment:
I'm not sure what bullet point to mark in bold as referencing this image (seems like a few related to ConvFit.
Can we maybe edit this image to remove the users fed id!

Reply:
I edited to remove the fed ID. the image is a generic one for the gui so there isnt a single point to make bold